plugins/modules/cloud/lxd/lxd_container.py

```diff
@@ -703,7 +729,10 @@ def main():
         ),
         supports_check_mode=False,
     )
-
+    if module.params['ignore_volatile_options']:
+        module.deprecate(
+            'The default behavior ignore the "volatile."-options. This will change in the future. Please test your scripts by "ignore_volatile_options: False"',
+            version='4.0.0', collection_name='community.general')
```

Comment:
Let's keep this commented out; to be uncommented in a follow-up PR that's not backported to stable-3.

Also I have some suggestions to improve this. Additionally to this, the default needs to be removed for that option (it needs to be re-added as `true` for 5.0.0), so that we can detect whether this option has been specified or not:
```suggestion
    # if module.params['ignore_volatile_options'] is None:
    #     module.params['ignore_volatile_options'] = False
    #     module.deprecate(
    #         'The default behavior is to ignore options starting with "volatile.". This will change in the future. Please test your playbooks by setting "ignore_volatile_options: false"',
    #         version='5.0.0', collection_name='community.general')
```

Reply:
done

Reply:
i commented this out but the proxmox module already use this.

Reply:
You still might want to copy the **other** changes from my suggestion (next to commenting out), since right now a) you would show this warning when the user specified this option (which is bad practice), b) `"volatile."-options` is a formulation which makes sense in German, but doesn't really work well in English.

(The deprecation warning in the proxmox module was added in c.g 1.3.0. In hindsight it should only have been added in 2.0.0, but it's too late to change that now. Let's do it better here.)

Reply:
I hope that i understand all your commets and fixed them. I saw to late that you already fixed the sentences of my denglish :) I tryed to find a better explanation for the users. Hope, that is better, at this try.

Reply:
(I just noticed that my suggestion should have said
```
    #     module.params['ignore_volatile_options'] = True
```
since the current default is `true` and not `false`.)